glide.yaml

```diff
@@ -32,4 +32,6 @@ import:
   version: ~1.1
 - package: github.com/arschles/assert
   version: 6882f85ccdc7c1822b146d1a6b0c2c48f91b5140
-- package: github.com/minio/minio-go
+- package: github.com/docker/distribution
```

Comment:
Ok, fine to leave as-is for now then.

Also, do you know the reasoning behind that regex? I'm particularly wondering if there's an object storage system that disallows `:` in object keys (I don't know of one)? Not blocking for this PR.